TTHAnalysis/python/tools/higgsDiffCompTTH.py

```diff
@@ -268,6 +325,10 @@ def analyze(self, event):
             
             self.out.fillBranch('%sdelR_lep_closest_wrongjet%s'%(self.label,jesLabel), leptonFromHiggs.DeltaR(j_closestWrong) if j_closestWrong else -99.)
 
+            # Other quantities
+            self.out.fillBranch('%smHrightlep%s'%(self.label,jesLabel)  ,  mHrightlep)
+            self.out.fillBranch('%smHwronglep%s'%(self.label,jesLabel)  ,  mHwronglep)
+
```

Comment:
Please remove the newline

Reply:
Which newline? The one before "# Other quantities" or the one after "self.out.fillB..."? BTW what is the rule that central wants followed w.r.t. this?

Reply:
The one after.
Bah, the general rule is that newlines shouldn't be added if not strictly necessary, which I agree is somehow arbitrary.
So far central enforces more or less the concept that almost any newline is a nuisance (because it makes the diffs grow larger, and in a big project with tens of thousands of commits this can go out of hand). I personally favour newlines to highlight conceptual blocks, but historically only newlines e.g. before the name of new functions are accepted by central.